dags/taar_weekly.py

```diff
@@ -10,7 +10,7 @@
 default_args_weekly = {
     "owner": "vng@mozilla.com",
     "depends_on_past": False,
-    "start_date": datetime(2019, 5, 31),
+    "start_date": datetime(2019, 7, 27),
```

Comment:
I would suggest setting this to 2019-07-14, so we can run this once it makes it into the production cluster.

Reply:
Changed as per your comment now.